src/components.js

```diff
@@ -826,14 +827,14 @@ const components = [
     name: 'Fixed',
     type: 'div',
     props: {},
-    style: props => ({
+    style: props => Object.assign({
       position: 'fixed',
       top: props.top ? 0 : null,
       right: props.right ? 0 : null,
       bottom: props.bottom ? 0 : null,
       left: props.left ? 0 : null,
       zIndex: props.z
-    }),
+    }, center(props)),
```

Comment:
After some looking around `components.js` and `utils.js`, I think that extracting logic into `utils.js` is not a good idea. Because the function `center` and other functions that tied with positioning components `Absolute` and `Fixed` is not useful for the other components. If you wish to extract the logic for other components, this idea, probably, would be relevant too. 
On the other hand a `components.js` now quite huge and would be bigger in future, when the number of components will increase, I hope :smile:  
So, what you think about splitting `components.js` into separate files in `components` folder? Those files can contain grouped components and their logic, for example `Positioning`, `Text`, `Buttons`. All those components would be joined in the `index.js` and exported as an array, like its doing now in `components.js`.